frontend:build fails

I get all the way to the point in the build process where I get a failure doing ./tash.sh frontend:build

     [exec] 
     [exec] > boston@1.0.0 build:boston /Users/french/Sites/devdesktop/boston.gov
     [exec] > gulp
     [exec] 
     [exec] [15:43:19] Using gulpfile ~/Sites/devdesktop/boston.gov/gulpfile.js
     [exec] [15:43:19] Starting 'clean:css'...
     [exec] [15:43:19] Finished 'clean:css' after 5.92 ms
     [exec] [15:43:19] Starting 'styles:production'...
     [exec] [15:43:19] Starting 'images'...
     [exec] [15:43:20] Starting 'clean:styleguide'...
     [exec] [15:43:20] Finished 'clean:styleguide' after 1.47 ms
     [exec] [15:43:20] Starting 'styleguide:markup'...
     [exec] [15:43:20] Finished 'styleguide:markup' after 11 ms
     [exec] [15:43:20] Starting 'styles:styleguide'...
     [exec] [15:43:20] Starting 'scripts'...
     [exec] [15:43:20] Starting 'scripts:iframe'...
     [exec] [15:43:20] Starting 'ie'...
     [exec] ./docroot/sites/all/themes/custom/boston/dist/ie
     [exec] [15:43:20] optimized images: docroot/sites/all/themes/custom/boston/src/img/b-logo-dark.png
     [exec] [15:43:20] optimized images: docroot/sites/all/themes/custom/boston/src/img/b-logo-large.png
     [exec] 
     [exec] events.js:141
     [exec]       throw er; // Unhandled 'error' event
     [exec]       ^
     [exec] Error: spawn /Users/french/Sites/devdesktop/boston.gov/node_modules/optipng-bin/vendor/optipng ENOENT
     [exec]     at exports._errnoException (util.js:856:11)
     [exec]     at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
     [exec]     at onErrorNT (internal/child_process.js:344:16)
     [exec]     at nextTickCallbackWith2Args (node.js:474:9)
     [exec]     at process._tickCallback (node.js:388:17)
     [exec] WARNING: In order to avoid variable namespace collisions, we have updated the way to add breakpoints for respond-to. Please change all instances of `$breakpoints: add-breakpoint()` to `@include add-breakpoint()`. The `add-breakpoint()` function will be deprecated in a future release.
     [exec] 	on line 12 of node_modules/breakpoint-sass/stylesheets/breakpoint/_respond-to.scss
     [exec] 	from line 39 of docroot/sites/all/themes/custom/boston/src/sass/components/button/_button.scss
     [exec] 
  [foreach] /Users/french/Sites/devdesktop/boston.gov/build/custom/phing/tasks/frontend.xml:6:44: Task exited with code 139

BUILD FAILED
/Users/french/Sites/devdesktop/boston.gov/build/custom/phing/tasks/frontend.xml:3:31: Execution of the target buildfile failed. Aborting.

Total time: 42.2654 seconds

--- a/guides/02-setting-up-development.md
+++ b/guides/02-setting-up-development.md
@@ -2,9 +2,10 @@
 
 ## Prereqs
 * Composer 1.1.2 [See documentation here](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx)
-* NVM 0.31 - `curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | bash`
-* node 5.4.0 - `source ~/.bashrc; nvm install 5.4.0; nvm use 5.4.0`
+* NVM - `curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | bash`
+* node (at least version 5.4.0) - `source ~/.bashrc; nvm install 5.4.0; nvm use 5.4.0`
 * Acquia Dev Desktop - we currently use Dev Desktop as our lamp stack. In the future, we'll have a VM to make this easier.
+* For gulp-image, you'll need libjpeg or libpng. Take a look at [gulp-image documentation](https://www.npmjs.com/package/gulp-image) for installation help.
 
 ## Initial repo setup
 * Fork the repository to your local github account.

